netlify-cms-academic/static/admin/index.html

```diff
@@ -11,5 +11,8 @@
 <body>
 <!-- Include the latest Netlify CMS v2.x.x script that builds the admin panel. -->
 <script src="https://cdn.jsdelivr.net/npm/netlify-cms@2/dist/netlify-cms.min.js"></script>
+<script src="hugo_shortcodes_netlify_cms.js"></script>
+<script type="module" src="preview.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/marked/1.2.3/marked.min.js" integrity="sha512-ZCO24NbTkJ7msQiCAkiufwqkuk9gcObBgDfBFa3F9k5mjGvmsfb/ben7s28Ss10xsbl3Pt+B5QqdH5Tg4V0Vag==" crossorigin="anonymous"></script>
```

Comment:
@paazca Netlify CMS appears to use the Remark Markdown parser: https://github.com/remarkjs/remark

So I'm curious (a) why you've chosen to use the _Marked_ Markdown parser, and (b) can we not reuse the existing Remark bundled in `netlify-cms.min.js` rather than loading an additional Markdown parser?

Reply:
I wasn't able to figure out how to render the markdown using Remark, so I used Marked as a temporary workaround. Hopefully someone else might be able to help with using remark,